packages/paste-core/components/heading/src/index.tsx

```diff
@@ -10,66 +11,86 @@ export type HeadingStyle =
   | 'headingStyle50'
   | 'headingStyle60';
 
-interface HeadingProps {
+interface Heading {
   as: asTags;
-  headingStyle?: HeadingStyle;
+  id?: never;
+  className?: never;
+  children: NonNullable<React.ReactNode>;
+  variant?: HeadingStyle;
 }
 
 function getHeadingStyles(headingStyle?: HeadingStyle): {} {
   switch (headingStyle) {
     case 'headingStyle60':
       return {
+        marginBottom: 'space70',
         fontSize: 'fontSize90',
-        fontWeight: 'fontWeightNormal',
-        lineHeight: 'lineHeight70',
+        fontWeight: 'fontWeightSemibold',
+        lineHeight: 'lineHeight60',
       };
     case 'headingStyle50':
+    default:
       return {
-        fontSize: 'fontSize80',
-        fontWeight: 'fontWeightNormal',
-        lineHeight: 'lineHeight40',
+        marginBottom: 'space60',
+        fontSize: 'fontSize70',
+        fontWeight: 'fontWeightSemibold',
+        lineHeight: 'lineHeight50',
       };
     case 'headingStyle40':
       return {
-        fontSize: 'fontSize70',
-        fontWeight: 'fontWeightSemiBold',
-        lineHeight: 'lineHeight40',
+        marginBottom: 'space50',
+        fontSize: 'fontSize60',
+        fontWeight: 'fontWeightSemibold',
+        lineHeight: 'lineHeight70',
       };
     case 'headingStyle30':
       return {
-        fontSize: 'fontSize60',
-        fontWeight: 'fontWeightSemiBold',
-        lineHeight: 'lineHeight30',
+        marginBottom: 'space40',
+        fontSize: 'fontSize40',
+        fontWeight: 'fontWeightSemibold',
+        lineHeight: 'lineHeight50',
       };
     case 'headingStyle20':
       return {
-        fontSize: 'fontSize50',
-        fontWeight: 'fontWeightSemiBold',
-        lineHeight: 'lineHeight30',
-      };
-    case 'headingStyle10':
-      return {
+        marginBottom: 'space30',
         fontSize: 'fontSize30',
-        fontWeight: 'fontWeightSemiBold',
+        fontWeight: 'fontWeightSemibold',
         lineHeight: 'lineHeight20',
       };
-    default:
+    case 'headingStyle10':
```

Comment:
Did you forget to add the default case here?

Reply:
Doesn't this say that `case 'headingStyle50` is also the `default`?

```
case 'headingStyle50':
default:
  return {
    marginBottom: 'space60',
    fontSize: 'fontSize70',
    fontWeight: 'fontWeightSemibold',
    lineHeight: 'lineHeight50',
  };
```

Reply:
I tried removing the default, but that causes a lint error (default required). For now I'm moving `headingStyle50` and `default` to the bottom, and adding a comment about why the heading styles are out of order.